src/engine/sparqlExpressions/SparqlExpression.cpp

```diff
@@ -108,6 +108,13 @@ Estimates SparqlExpression::getEstimatesForFilterExpression(
   return {inputSizeEstimate, inputSizeEstimate};
 }
 
+// _____________________________________________________________________________
+std::vector<PrefilterExprVariablePair>
+SparqlExpression::getPrefilterExpressionForMetadata(
+    [[maybe_unused]] bool isNegated) const {
```

Comment:
Add a short comment (default implementation, no supported prefiltering, has to be explicitly overridden to be supported, etc.